ONSdigital/eq-author-app
# Added UNIT type to routing

### What is the context of this PR?
Added Unit types to routing.

### How to review 
1. Create answer with unit type.
2. Ensure routing displays the correct unit type when unit type answer is selected
3. Ensure routing works with unit type as expected
